docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -79,7 +83,10 @@ Property Name                                              Description
 ``kafka.ssl.truststore.type``                              File format of the truststore file, defaults to ``JKS``
 ``kafka.ssl.key.password``                                 Password for the private key in the keystore file
 ``kafka.ssl.endpoint-identification-algorithm``            Endpoint identification algorithm used by clients to validate server host name, defaults to ``https``
-========================================================== ==============================================================================
+``kafka.config.resource``                                  Optional configuration file of Kafka client. These files must exist on the
```

Comment:
Please make it coherent with: https://trino.io/docs/current/connector/hive.html#hive-configuration-properties